tests/src/core/testqgsgdalprovider.cpp

```diff
@@ -81,6 +83,9 @@ void TestQgsGdalProvider::initTestCase()
 
   mTestDataDir = QStringLiteral( TEST_DATA_DIR ) + '/'; //defined in CmakeLists.txt
   mReport = QStringLiteral( "<h1>GDAL Provider Tests</h1>\n" );
+
+  mSupportsNetCDF = GDALGetDriverByName( "netcdf" ) != NULL;
```

Comment:
```suggestion
  mSupportsNetCDF = static_cast< bool >( GDALGetDriverByName( "netcdf" ) );
```

a touch more modern :wink: